models/project/project.go

```diff
@@ -309,6 +317,19 @@ func (u *ProjectStruct) FetchGithub() (bool, error) {
 	return true, nil
 }
 
+func (u *ProjectStruct) CreateWebhook() error {
```

Comment:
__[golint]__
* [golint] exported method ProjectStruct.CreateWebhook should have comment or be unexported